client/public/elements/utils/rp-utils-collection.js

```diff
@@ -370,6 +370,13 @@ export default class RpUtilsCollection extends Mixin(LitElement)
       q.subFacet = this.subFacet;
     }
 
+    const textSearchFields = localStorage.getItem('textSearchFields');
```

Comment:
Does this actually propagate the input fields into the query?

Reply:
Yea, the _textSearchFields_ is storing the defaults/concepts/function values, so it basically is checking if the users local storage has an override and uses it if it does.

Reply:
It doesn't override index.js, just if it exists in localStorage that's used over index.js pretty much.